downstream/modules/platform/con-pod-specification-mods.adoc

```diff
@@ -0,0 +1,106 @@
+The labels are stored in key:value format in the metadata hash. 
+One label in this example is `docker-registry=default`.
+| `generateName:` | Pods must have a unique name within their namespace. 
+A pod definition can specify the basis of a name with the `generateName` attribute, and add random characters automatically to generate a unique name.
+| `containers:` | `containers` specifies an array of container definitions. In this case (as with most), defines only one container.
+| `env:` | Environment variables pass necessary values to each container.
+| `image:` | Each container in the pod is instantiated from its own Docker-formatted container image.
+| `ports:` |  The container can bind to ports made available on the pod’s IP.
+| `resources` | When you specify a Pod, you can optionally describe how much of each resource a container needs. 
+The most common resources to specify are CPU and memory (RAM). 
+Other resources are available.
+| `securityContext` | OpenShift Online defines a security context for containers that specifies whether they are permitted to run as privileged containers, run as a user of their choice, and more. 
+The default context is very restrictive but administrators can modify this as required.
+| `volumeMounts:` | The container specifies where external storage volumes should be mounted within the container. 
+In this case, there is a volume for storing the registry’s data, and one for access to credentials the registry needs for making requests against the OpenShift Online API.
+| `ImagePullSecrets` | A pod can contain one or more containers, which must be pulled from some registry. 
+If containers come from registries that require authentication, you can provide a list of `ImagePullSecrets` that refer to `ImagePullSecrets` present in the namespace. 
+Having these specified enables {OCP} to authenticate with the container registry when pulling the image. 
+For further information, see link:https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/[Managing resource containers] in the kubernetes documentation.
+| `restartPolicy:` | The pod restart policy with possible values `Always`, `OnFailure`, and `Never`. 
+The default value is `Always`.
+| `serviceAccount:` | Pods making requests against the OpenShift Online API is a common enough pattern that there is a `serviceAccount` field for specifying which service account user the pod should authenticate as when making the requests. This enables fine-grained access control for custom infrastructure components.
+| `volumes:` | The pod defines storage volumes that are available to its container(s) to use. 
+In this case, it provides an ephemeral volume for the registry storage and a secret volume containing the service account credentials.
+|====
+
+You can  modify the pod used to run jobs in a kubernetes-based cluster using {ControlerNameShort} by editing the pod specification in the {ControllerNameShort} UI.  
```

Comment:
```suggestion
You can  modify the pod used to run jobs in a kubernetes-based cluster using {ControllerNameShort} by editing the pod specification in the {ControllerNameShort} UI.  
```

The 2nd instance of the controller attribute isn't rendering properly:
![image](https://user-images.githubusercontent.com/44700011/201900475-39761028-a75e-4e5c-a15a-63a7323cddbb.png)